CHANGELOG.md

```diff
@@ -148,5 +175,7 @@
 - Confine rspec pinning to ruby 1.8
 
 
-
 \* *This Change Log was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
+
+
+\* *This Changelog was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
```

Comment:
That's two footers :wink:

Reply:
fixed

Reply:
@bastelfreak  https://github.com/camptocamp/puppet-systemd/pull/78